src/components/ReferenceSolutions/ReferenceSolutionStatus/ReferenceSolutionStatus.js

```diff
@@ -8,11 +8,12 @@ import DateTime from '../../widgets/DateTime';
 import UsersNameContainer from '../../../containers/UsersNameContainer';
 import ExercisesNameContainer from '../../../containers/ExercisesNameContainer';
 import Icon, { EditIcon, CodeIcon } from '../../icons';
+import EnvironmentsListItem from '../../helpers/EnvironmentsList/EnvironmentsListItem';
 
-const ReferenceSolutionStatus = ({ description, solution: { userId, createdAt }, exerciseId }) => (
+const ReferenceSolutionStatus = ({ description, userId, submittedAt, submittedBy, exerciseId, environment }) => (
```

Comment:
This comes with any API change?

Reply:
No, the solution structure was created in component above...